docs/investigations/api-documentation/generate-api-documentation.md

```diff
@@ -0,0 +1,59 @@
+# Generating API documentation
+
+## Overview
+
+This document discusses how to create a static documentation page from our GraphQL schema.
+
+As of right now, there is no static page with our API documentation, so if any problem occurs the user can either check the `examples` directory or read the schema on his own.
+We need a solution that would be available on one of the Compass endpoints and would utilise our generated examples.
+
```

Comment:
Add "Requirements" section, or some "Assumptions" - what we want to achieve, what is needed

something like that examples need to be synced with specific Director version, etc.